Makefile

```diff
@@ -3,7 +3,10 @@ BIN = dive
 all: clean build
 
 run: build
-	./build/$(BIN) build -t dive-test:latest -f .data/Dockerfile .
+	./build/$(BIN) build -t dive-example:latest -f .data/Dockerfile.example .
+
+run-ci: build
+	CI=true ./build/$(BIN) build -t dive-example:latest -f .data/Dockerfile.example .
```

Comment:
Can we add `--ci-config ./.data/.dive-ci`?

The default rules make sense, but I think the defile behavior of `make run-ci` should exercise all the rules, and not skip the wasted bytes rule.